nevergrad/functions/gym/gym_multi.py

```diff
@@ -0,0 +1,180 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+import numpy as np
+import os
+import gym
+
+if os.name != "nt":
+    import gym_anm  # type: ignore  # pylint: disable=unused-import
+from nevergrad.parametrization import parameter
+from ..base import ExperimentFunction
+
+
+# Method for building a new list, for a future version of gym:
+#
+# gym_env_names = ["gym_anm:ANM6Easy-v0"]
+#
+# for e in gym.envs.registry.all():
+#     try:
+#         assert "Kelly" not in e.id
+#         env = gym.make(e.id)
+#         a1 = env.action_space.sample()
+#         a2 = env.action_space.sample()
+#         a3 = env.action_space.sample()
+#         a1 = a1 + a2 + a3
+#         if hasattr(a1, "size"):
+#             assert a1.size() < 15000
+#         gym_env_names.append(e.id)
+#     except:
+#         pass
+
+gym_env_names = [
```

Comment:
prefer capitalized names for global variables
However, is there no way to actually fetch this information within the package? 

```suggestion
GYM_ENV_NAMES = [
```

Reply:
Done.